### PR TITLE
Update LinuxInstall.md

### PR DESCRIPTION
**Pull request by wthomson**
Prompt user to create the html directory if it doesn't exist.

### CODE REVIEW

**Comment by Paul-Reed** [comment]
Isn't the html directory automatically created by apache2 
Under what circumstance would it not exist if the installation guide was followed?

Paul

**Comment by wthomson** [comment]
I thought so too. But when I installed Apache 2.2, there was no html directory.

**Comment by glynhudson** [comment]
It's always been created for me. What system did you install apache on? Is this the  latest version of apache?
 Maybe you could add a note to create the directory if it doesn't exist?

**Comment by wthomson** [comment]
I *did* include a line that says:
If you get an error message "No such file or directory" you'll need to create the directory,
It's not the latest version. I did a apt-get install apache2 during a manual build and it gave me version 2.2

File: docs/LinuxInstall.md [hunk 1]
```diff
@@ -63,12 +63,21 @@ First cd into the /var/www directory:
...
+    sudo mkdir html
+    
+then change the owner:
 
     sudo chown $USER html
 
-Cd into html directory
+Change into the html directory. 
 
+    sudo mkdir html
```
Comment (Paul-Reed) [inline docs/LinuxInstall.md]: The html directory will have already been created either by apache or by L72.
No need to create it again.

**Comment by wthomson** [comment]
Weird. It wasn't there when I did the manual build. When I went to change ownership of the file it gave me an error saying the file/directory did not exist.

PR closed.

**Comment by Paul-Reed** [comment]
As Glyn said above, there is no harm in adding a note to create if it doesn't exist, but your PR contains a typo where you run `$ sudo mkdir html` twice.
Once on line 72 and again on line 80.

If you wish to amend, I'll merge your PR into the guide as a safeguard.

Paul

**Pull request closed without merging**